python_modules/automation/automation/docker/images/dagster-base/Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE
+FROM "${BASE_IMAGE}"
+
+ARG DAGSTER_VERSION
+
+# ==> Dagster base layer
+RUN \
+# Cron - remove along with dagster-cron once daemon scheduler is default
+       apt-get update -yqq \
+    && apt-get install -yqq --no-install-recommends cron \
+# Dagster
+    && pip install \
```

Comment:
Could using the flag `pip install --no-cache-dir ...` allow us to remove the `rm -rf /root/.cache` command down below? If so, then I think this would be preferable and more clear.

Docs on `--no-cache-dir`: https://pip.pypa.io/en/stable/reference/pip_install/#caching

Reply:
Confirming - `--no-cache-dir` removes the need to `rm -rf /root/.cache`